### DISCUSSION TITLE
Change the font size

### DISCUSSION THREAD

**Discussion by danzecch** [YukiGasai/obsidian-google-calendar#56] [Q&A] [answered]
Hi, how can I change the font size of the events in the Timeline View?

**Comment by YukiGasai** [comment, marked as answer]
You can change the appearance using CSS snippets inside the obsidian settings.
There are several options to target the text inside events with CSS classes.

```
googleCalendarEvent (For all events)
googleCalendarEvent_Calendar_Color_{Calender-Color-Id}
googleCalendarEvent_Event_Color_{Event-Color-Id}
googleCalendarEvent_Id_{Event-Id}
```

**Reply by danzecch** [reply]
Thanks.
So I tried to do this:
```
.googleCalendarEvent{
font-size: 50px;
}
```
But nothing changes.

The name of this CSS file is relevant?

**Reply by YukiGasai** [reply]
Hi,
sorry for the delay. The name of the CSS file is not relevant. 

The `googleCalendarEvent` is for the entire div, use `googleCalendarName` for just the text. Then it should work.

```css
.googleCalendarName{
font-size: 30px;
}
```

Plus. Make sure to enable the css sheet in the obsidian settings.
![image](https://user-images.githubusercontent.com/38146192/217877888-fca0bb3d-7f83-48cf-bd39-91871ca3bfb5.png)

**Reply by danzecch** [reply]
Thanks
